### DISCUSSION TITLE
Active directory

### DISCUSSION THREAD

**Discussion by moayadsquad** [directus/directus#12500] [Archive]
Does directus support login via active directory?

**Comment by aidenfoxx** [comment]
Yes, via the LDAP driver. See the docs: https://docs.directus.io/configuration/config-options/#ldap-ldap

**Reply by rijkvanzanten** [reply]
And depending on which AD service you use, you might also be able to utilize OpenID or OAuth2 (as is the case with MS' Azure Active Directory)
